krux_boto/boto.py

```diff
@@ -216,3 +256,51 @@ def wrapper(*args, **kwargs):
             return wrapper
 
         return attr
+
+class Boto(BaseBoto):
+
+    # All the hard work is done in the superclass. We just need to use the
+    # resulting object to initialize a session properly.
+    def __init__(self, *args, **kwargs):
```

Comment:
They are written out in BaseBoto's init, and are simply being passed on verbatim. Writing them out would mean keeping 3 places in sync, for no benefit. I feel the comment captures the intent, but I'd be happy to clarify differently if you have any suggestions.